packages/ui-extensions/src/surfaces/admin/components/patterns/examples/setupGuide.html

```diff
@@ -0,0 +1,161 @@
+                  Select a template for your puzzle - choose between 9-piece (beginner),
+                  16-piece (intermediate), or 25-piece (advanced) layouts.
+                </s-paragraph>
+                <s-stack direction="inline" gap="small-200">
+                  <s-button variant="primary">Choose template</s-button>
+                  <s-button variant="tertiary" tone="neutral"> See all templates </s-button>
+                </s-stack>
+              </s-grid>
+              <s-box maxBlockSize="80px" maxInlineSize="80px">
+                <s-image
+                  src="https://cdn.shopify.com/s/assets/admin/checkout/settings-customizecart-705f57c725ac05be5a34ec20c05b94298cb8afd10aac7bd9c7ad02030f48cfa0.svg"
+                  alt="Customize checkout illustration"
+                ></s-image>
+              </s-box>
+            </s-grid>
+          </s-box>
+        </s-box>
+      </s-box>
+      <!-- Step 3 -->
+      <s-divider></s-divider>
+      <s-box>
+        <s-box padding="small">
+          <s-grid gridTemplateColumns="1fr auto" gap="base">
+            <s-checkbox
+              label="Customize puzzle piece shapes"
+              onInput="window.puzzleApp.updateProgress(this, document.getElementById('progress-text'))"
+            ></s-checkbox>
+            <s-button
+              id="toggle-step3-button"
+              onClick="window.puzzleApp.toggleStep(this, document.getElementById('step3-details'))"
+              accessibilityLabel="Toggle step 3 details"
+              variant="tertiary"
+              icon="chevron-down"
+            ></s-button>
+          </s-grid>
+        </s-box>
+        <s-box id="step3-details" padding="small" paddingBlockStart="none" style="display: none;">
+          <s-box padding="base" background="subdued" borderRadius="base">
+            <s-grid gridTemplateColumns="1fr auto" gap="base" alignItems="center">
+              <s-grid gap="small-200">
+                <s-paragraph>
+                  Make your puzzle unique by customizing the shapes of individual pieces.
+                  Choose from classic, curved, or themed piece styles.
+                </s-paragraph>
+                <s-stack direction="inline" gap="small-200">
+                  <s-button variant="primary"> Customize pieces </s-button>
+                  <s-button variant="tertiary" tone="neutral">
+                    Learn about piece styles
+                  </s-button>
+                </s-stack>
+              </s-grid>
+              <s-box maxBlockSize="80px" maxInlineSize="80px">
+                <s-image
+                  src="https://cdn.shopify.com/s/assets/admin/checkout/settings-customizecart-705f57c725ac05be5a34ec20c05b94298cb8afd10aac7bd9c7ad02030f48cfa0.svg"
+                  alt="Customize checkout illustration"
+                ></s-image>
+              </s-box>
+            </s-grid>
+          </s-box>
+        </s-box>
+      </s-box>
+      <!-- Add additional steps here... -->
```

Comment:
All of these comments don't render right for me when I copy this into code sandbox, we should remove or change the format